lib/color_mask.py

```diff
@@ -12,20 +14,23 @@ def get_frame(cap, scaling_factor):
 
     return frame
 
-if __name__=='__main__':
+def run_mask():
     cap = cv2.VideoCapture(0)
     scaling_factor = 0.5
 
     # Iterate until the user presses ESC key
     while True:
         frame = get_frame(cap, scaling_factor)
-
+        hsv_max = colors.hsv_max(frame)
         # Convert the HSV colorspace
         hsv = cv2.cvtColor(frame, cv2.COLOR_BGR2HSV)
 
         # Define 'blue' range in HSV colorspace
-        lower = np.array([60,100,100])
-        upper = np.array([180,255,255])
+        #lower = np.array([60,100,100])
+        #upper = np.array([180,255,255])
+
+        lower = np.array([hsv_max[0]-5,hsv_max[1]-20,hsv_max[2]-20])
+        upper = np.array([hsv_max[0]+5,hsv_max[1]+20,hsv_max[2]+20])
```

Comment:
Having numbers like `20` and `5` are _magic_ ... in other words, without really understanding the code (and the behavior of underlying functions), these values just _work magic_. One way to help with this, is to convert the magic numbers to constants with appropriate names. In this case, we might want to add a comment about how we arrived at the use of `20` for the third value, but not the first.